backend/test/forgotPassword.test.js

```diff
@@ -0,0 +1,103 @@
+const { expect } = require('chai');
+const chaiHttp = require('chai-http');
+const chai = require('chai');
+const Admin = require('../app/models/Admin');
+const server = require('../index');
+
+chai.use(chaiHttp);
+
+before((done) => {
+  // Removing the test admin data before running all the test cases
+  Admin.remove({ email: 'john@wick.com' }, () => {
```

Comment:
this should ideally be the last step,
You should clear the resource once you are done with the tests and not before starting any test.
Ideally there shouldn't be that particular resource before the test so what's the point in removing? This should be the last step